spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -1547,7 +1548,8 @@ protected Table createTablePartitioned(
   }
 
   protected Table createTablePartitioned(int partitions, int files) {
-    return createTablePartitioned(partitions, files, SCALE, Maps.newHashMap());
+    Map<String, String> properties = ImmutableMap.of(TableProperties.FORMAT_VERSION, "1");
```

Comment:
Why V1 For Rewrite DataFiles?  (also just this particular create method?)

Reply:
I can change that for a single test, actually. I'll update.

Reply:
Updated.